core/src/lib/code-generation/code-builder/code-builder-imports.ts

```diff
@@ -0,0 +1,37 @@
+import { IQueueStep } from './queued-code-builder/queue-step';
+import { CodeBuilder } from './code-builder';
+
+type ILibraryImport = {
+    from: string;
+    elements: {
+        element: string;
+        importAs?: string;
+    }[];
+};
+
+export class CodeBuilderImports implements IQueueStep {
+
+    private imports: ILibraryImport[] = [];
+
+    add(element: string, from: string, importAs?: string): void {
```

Comment:
Fände folgenden Kommentar gut entweder hier oder bei der Definition von ILibraryImport

// import { <element> } as <importAs> from <from>

Dan sieht man sofort auf einen Blick wofür alles steht

Reply:
This is already declared in the QuedCodeBuilder. Which is the public api for managing imports.